OWML.Patcher/OWPatcher.cs

```diff
@@ -31,10 +31,20 @@ public void PatchGame()
 
         private void CopyFiles()
         {
-            var filesToCopy = new[] { "OWML.ModLoader.dll", "OWML.Common.dll", "OWML.ModHelper.dll",
-                "OWML.ModHelper.Events.dll", "OWML.ModHelper.Assets.dll", "OWML.ModHelper.Menus.dll",
-                "Newtonsoft.Json.dll", "System.Runtime.Serialization.dll", "0Harmony.dll", "NAudio-Unity.dll",
-                "OWML.Manifest.json" };
+            var filesToCopy = new[] {
+                "OWML.ModLoader.dll",
+                "OWML.Common.dll",
+                "OWML.ModHelper.dll",
+                "OWML.ModHelper.Events.dll",
+                "OWML.ModHelper.Assets.dll",
+                "OWML.ModHelper.Menus.dll",
+                "OWML.ModHelper.Logging.dll",
+                "Newtonsoft.Json.dll",
+                "System.Runtime.Serialization.dll",
+                "0Harmony.dll",
+                "NAudio-Unity.dll",
+                "OWML.Manifest.json"
+            };
```

Comment:
Hope you don't mind me making these vertical, I think it makes a lot more sense like this since it makes it a lot easier to add / remove items.

Reply:
I anti-mind it 🎖️